src/zypper_plugins/tracer_upload.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
+#
+# Copyright 2019 ATIX AG
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+
+import json
+import httplib
+from os import path, environ
+import sys
+import logging
+from rhsm.config import RhsmConfigParser, initConfig
+from zypp_plugin import Plugin
+
+sys.path.append('/usr/share/rhsm')
+from subscription_manager.identity import ConsumerIdentity
+
+# The path is defined in zypper, see https://github.com/openSUSE/libzypp/blob/master/zypp/target/TargetImpl.cc
+REBOOT_NEEDED_FLAG = "/var/run/reboot-needed"
+
+class TracerUploadPlugin(Plugin):
+    def __init__(self):
+        Plugin.__init__(self)
```

Comment:
Should this use `super(TracerUploadPlugin, self).__init__()`

Reply:
Had a look at other Plugins and removed the def __init__(self) method completely.